### DISCUSSION TITLE
blog/mindmap-review/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#362] [Announcements]
# blog/mindmap-review/

探索本地和在线思维导图优劣，揭秘为什么我最终选择了 Whimsical。

https://jimmysong.io/blog/mindmap-review/

**Comment by kelvinji2009** [comment]
Whimsical确实非常好用

**Reply by rootsongjc** [reply]
比较遗憾的是，Whimsical 不支持文本导入和无损格式导出，这就不太方便了，只适合在网页中嵌入或者直接链接访问了。
